### DISCUSSION TITLE
How to use Toolbox

### DISCUSSION THREAD

**Discussion by buithehien1991** [AllYourBot/hostedgpt#561] [General]
Dear @krschacht 

Could you please tell me how to using toolbox or any guide for using it?

I want to use Memory toolbox but don't know how to get it. 

Thank you so much and best regards,

**Comment by keithschacht** [comment]
It just works automatically. Toolbox is the class name which defines functions for the LLM "function calling" or "tool use", those are two names for the same thing. You have to use an assistant which supports this and currently it's only the OpenAI ones.

Try telling the LLM "My favorite food is pizza" and then go to settings and check memory, you'll see it has now remembered that.

**Reply by buithehien1991** [reply]
I understood, thank you so much.
